### ISSUE TITLE
Does it support Final IK？

### ISSUE THREAD

**Issue by cracker999999** [mikeNspired/XRI-Starter-Kit#29]

**Comment by mikeNspired** [comment]
No, I have zero plans on doing that. It's meant to just be a simple hand
poser.  Unless my day job starts using final IK, I might get the urge to
implement it into this.
